linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1643,76 +1680,1188 @@ where
+        .requested_proposed
+        .unwrap()
+        .content
+        .block
+        .operations
+        .contains(&Operation::System(SystemOperation::PublishBlob {
+            blob_id: blob3_id
+        })));
+    assert!(validator_manager
+        .requested_locked
+        .clone()
+        .unwrap()
+        .value()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&Operation::System(SystemOperation::ReadBlob {
+            blob_id: blob2_id
+        })));
+    assert!(validator_manager
+        .requested_locked
+        .unwrap()
+        .value()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&Operation::System(SystemOperation::PublishBlob {
+            blob_id: blob3_id
+        })));
+
+    builder.set_fault_type([1], FaultType::Offline).await;
+    builder.set_fault_type([0, 2, 3], FaultType::Honest).await;
+
+    client4.synchronize_from_validators().await.unwrap();
+    let bt_certificate = client4
+        .burn(None, Amount::from_tokens(1), UserData::default())
+        .await
+        .unwrap()
+        .unwrap();
+
+    let hashed_certificate_values = client4
+        .read_hashed_certificate_values_downward(bt_certificate.value.hash(), 4)
+        .await
+        .unwrap();
+
+    // Latest block should be the burn
+    assert!(hashed_certificate_values[0]
+        .inner()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&Operation::System(SystemOperation::Transfer {
+            owner: None,
+            recipient: Recipient::Burn,
+            amount: Amount::from_tokens(1),
+            user_data: UserData::default(),
+        })));
+
+    // Block before that should be b1
+    assert!(hashed_certificate_values[1]
+        .inner()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&Operation::System(SystemOperation::ReadBlob {
+            blob_id: blob2_id
+        })));
+    assert!(hashed_certificate_values[1]
+        .inner()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&Operation::System(SystemOperation::PublishBlob {
+            blob_id: blob3_id
+        })));
+
+    // Previous should be the `ChangeOwnership` operation
+    assert!(hashed_certificate_values[2]
+        .inner()
+        .block()
+        .unwrap()
+        .operations
+        .contains(&owner_change_op));
+    Ok(())
+}
+
+#[test_case(MemoryStorageBuilder::default(); "memory")]
+#[cfg_attr(feature = "storage-service", test_case(ServiceStorageBuilder::new().await; "storage_service"))]
+#[cfg_attr(feature = "rocksdb", test_case(RocksDbStorageBuilder::new().await; "rocks_db"))]
+#[cfg_attr(feature = "dynamodb", test_case(DynamoDbStorageBuilder::default(); "dynamo_db"))]
+#[cfg_attr(feature = "scylladb", test_case(ScyllaDbStorageBuilder::default(); "scylla_db"))]
+#[test_log::test(tokio::test)]
+async fn test_request_leader_timeout<B>(storage_builder: B) -> anyhow::Result<()>
```

Comment:
The diff shows this and `test_finalize_validated` as new (and the original ones as changed into the new blob tests); were these actually changed?

Reply:
Weird, on Graphite it doesn't show them as new for me 🤔
Might be GitHub's diff being buggy. They were not changed